app/src/components/search/content.tsx

```diff
@@ -0,0 +1,103 @@
+import {
+  Box,
+  Menu,
+  SimpleGrid,
+  Text,
+} from "@nypl/design-system-react-components";
+import { mockCollections } from "__tests__/__mocks__/data/mockCollections";
+import { useSearchParams } from "next/navigation";
+import { CollectionCardModel } from "../../models/collectionCard";
+import CollectionDataType from "../../types/CollectionDataType";
+import CollectionCard from "../cards/collectionCard";
+import { useNumColumns } from "../../hooks/useNumColumns";
+import useBreakpoints from "../../hooks/useBreakpoints";
+import { useRouter } from "next/navigation";
+
+const SearchContent = () => {
+  const queryParams = useSearchParams();
+  const query = queryParams.toString();
+  const numColumns = useNumColumns();
+  const { isLargerThanLargeTablet } = useBreakpoints();
+
+  const router = useRouter();
+
+  const createQueryString = (name, value) => {
```

Comment:
This will need a much fuller implementation– just a starting point to show how we can add sort/filter queries alongside search queries

Reply:
oh, I was under the impression that this page would support a max of two query params: one for search and one for filter.

kind of how the url on the old site already behaves: https://digitalcollections.nypl.org/collections?utf8=%E2%9C%93&collection_keywords=cat&sort=title 

but we should definitely double check with Lana.

Reply:
The sort and filter is so broken on old DC that i think this is definitely a good opportunity to reevaluate

Reply:
Wouldn't it need three? The search, the sort, and then the order?

Reply:
By order do you mean ascending/descending ? The implementation of that will have to change here because of how we're displaying the options– on old DC the order doesn't even appear in the URL, and it's pretty unclear what the up/down arrows even mean, but here the order is included in the menu options. Again, I didn't mean this to be the final implementation of the sort, I think that's something that will have to wait on the finished endpoint and more decisions about URL parity

Reply:
Ah yes, that's right. I keep thinking of legacy DC's query params. This would break parity but I think that's a good thing in this case.